src/pages/home/owners/OwnersTable.tsx

```diff
@@ -64,86 +68,102 @@ export default function OwnersTable({ ownersData }: OwnersTableProps) {
-                </StyledTableCell>
-                <StyledTableCell>{ row.song }</StyledTableCell>
-                <StyledTableCell>{ row.info_tbd }%</StyledTableCell>
-              </TableRow>
-            )) }
-          { emptyRows > 0 && (
-            <TableRow style={ { height: rowHeight * emptyRows } }>
-              <StyledTableCell
-                sx={ { backgroundColor: theme.colors.black100 } }
-                colSpan={ 6 }
-              />
-            </TableRow>
-          ) }
-        </TableBody>
-        <TableFooter
-          sx={ {
-            backgroundColor: theme.colors.grey600,
-          } }
-        >
-          <TableRow>
-            <StyledTableCell colSpan={ 3 }>
-              <Box
+  if (ownersData) {
+    return (
+      <TableContainer>
+        <Table size="small" aria-label="Owners pagination table">
+          <TableHead>
+            <TableRow>
+              <StyledTableCellHeader>
+                <Typography>OWNER</Typography>
+              </StyledTableCellHeader>
+              <StyledTableCellHeader>
+                <Typography>SONG</Typography>
+              </StyledTableCellHeader>
+              <StyledTableCellHeader
                 sx={ {
-                  display: "flex",
-                  justifyContent: "space-between",
-                  alignItems: "center",
+                  display: { xs: "none", sm: "inline" },
                 } }
               >
-                <Typography variant="body1" sx={ { fontWeight: 500 } }>
-                  Showing { (page - 1) * rowsPerPage + 1 } to { lastRowOnPage } of{ " " }
-                  { ownersData.length } owners
-                </Typography>
-                <StyledPagination
-                  variant="outlined"
-                  shape="rounded"
-                  page={ page }
-                  count={ Math.ceil(ownersData.length / rowsPerPage) }
-                  onChange={ handlePageChange }
+                <Typography>INFO TBD</Typography>
+              </StyledTableCellHeader>
+            </TableRow>
+          </TableHead>
+          <TableBody sx={ { backgroundColor: theme.colors.grey600 } }>
+            { ownersData
+              .slice(
+                (page - 1) * rowsPerPage,
+                (page - 1) * rowsPerPage + rowsPerPage
+              )
+              .map((row) => (
+                <TableRow key={ row.id }>
+                  <StyledTableCell
+                    sx={ {
+                      color: theme.colors.white,
+                      maxWidth: ["150px", "unset"],
+                    } }
+                  >
+                    <Box
+                      sx={ { display: "flex" } }
+                      overflow="scroll"
+                      whiteSpace="nowrap"
+                    >
+                      <Box>{ row.name }</Box>
+                      { row.registered ? (
+                        <Box
+                          sx={ {
+                            marginLeft: "15px",
```

Comment:
Action Needed: can this be 16 px, just so that we stay consistent with the 4px grid?

Reply:
✅